package.json

```diff
@@ -36,12 +34,15 @@
     "release-win": "npm run build-release -- -w --prepackaged release/windows --publish=always",
     "release-lin": "npm run build-release -- -l --prepackaged release/linux && NAME=$npm_package_name VERSION=$npm_package_version ./aptly.sh",
     "release": "npm run release-mac && npm run release-win && npm run release-lin",
-    "test-app": "mocha --recursive test/app",
-    "test-modules": "mocha --recursive test/modules",
-    "test-server": "mocha --recursive test/server",
-    "test-server-coverage": "nyc --reporter=lcov --reporter=text mocha --recursive test/server",
+    "test-node": "mocha --recursive test/app test/modules ts/test-unit",
+    "test-electron": "yarn grunt test",
+    "test-electron-coverage": "nyc --reporter=lcov --reporter=text mocha --recursive test/app test/modules ts/test-unit",
+    "test": "yarn test-node && yarn test-electron",
+    "jscs": "yarn grunt jscs",
+    "jshint": "yarn grunt jshint",
     "eslint": "eslint .",
-    "tslint": "tslint ts/**/*.{ts,tsx}",
+    "tslint": "tslint --config tslint.json --project . --format stylish",
+    "lint": "yarn eslint && yarn tslint && yarn jshint && yarn jscs",
```

Comment:
While this does result in good visual consistency across the four style tools, I don't think we want to spin up multiple instances of grunt. It's pretty heavyweight.

Reply:
Fair point. I can add `yarn grunt lint` to cover both.